### PR TITLE
[node] keyframeselection update with simpler algorithms

### PR DESCRIPTION
**Pull request by servantftechnicolor**
new keyframe selection algorithm

[See Alicevision PR](https://github.com/alicevision/AliceVision/pull/1213)

### CODE REVIEW

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 1]
```diff
@@ -168,6 +88,13 @@ class KeyframeSelection(desc.CommandLineNode):
             range=(0, 10000, 1),
             uid=[0],
         ),
+        desc.BoolParam(
+            name='useSimpleMethod',
+            label='Use simple method',
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:93]: Upper Camel Case for Labels as everywhere.

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 2]
```diff
@@ -168,6 +88,13 @@ class KeyframeSelection(desc.CommandLineNode):
             range=(0, 10000, 1),
             uid=[0],
         ),
+        desc.BoolParam(
+            name='useSimpleMethod',
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:92]:
```suggestion
            name='useRegularKeyframes',
```

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 3]
```diff
@@ -65,84 +65,12 @@ class KeyframeSelection(desc.CommandLineNode):
...
-            uid=[0],
-        ),
-        desc.IntParam(
-            name='sharpSubset',
-            label='Sharp Subset',
-            description='''sharp part of the image (1 = all, 2 = size/2, ...)''',
-            value=4,
-            range=(1, 100, 1),
-            uid=[0],
+            uid=[],
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:73]:
```suggestion
            uid=[0],
```

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 4]
```diff
@@ -168,6 +88,13 @@ class KeyframeSelection(desc.CommandLineNode):
             range=(0, 10000, 1),
             uid=[0],
         ),
+        desc.BoolParam(
+            name='useSimpleMethod',
+            label='Use simple method',
+            description="Disable smart algorithm for frame selection",
+            value=False,
+            uid=[],
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:96]:
```suggestion
            uid=[0],
```

File: meshroom/nodes/aliceVision/KeyframeSelection.py [hunk 5]
```diff
@@ -65,84 +65,12 @@ class KeyframeSelection(desc.CommandLineNode):
             label="mmFocals",
             description="Focals in mm (will be use if not 0)."
         ),
-        desc.ListAttribute(
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/KeyframeSelection.py:68]: We need to generate images with the right metadata.

Comment (servantftechnicolor) [reply meshroom/nodes/aliceVision/KeyframeSelection.py:68]: I do exactly what was done before ? Except we don't use anymore focal in pixels.

File: meshroom/nodes/aliceVision/KeyframeSelection.py (hunk 1 above)
Comment (cbentejac) [reply meshroom/nodes/aliceVision/KeyframeSelection.py]: Done.

File: meshroom/nodes/aliceVision/KeyframeSelection.py (hunk 4 above)
Comment (cbentejac) [reply meshroom/nodes/aliceVision/KeyframeSelection.py]: Done.

File: meshroom/nodes/aliceVision/KeyframeSelection.py (hunk 2 above)
Comment (cbentejac) [reply meshroom/nodes/aliceVision/KeyframeSelection.py]: Done.

**Comment by cbentejac** [comment]
Replaced by #1880.

**Pull request closed without merging**